packages/frontend-2/lib/projects/composables/projectManagement.ts

```diff
@@ -135,69 +129,7 @@ export function useCreateProject() {
           : {
               mutation: createProjectMutation,
               variables: { input }
-            }),
-        update: (cache, { data }) => {
-          // not sure why this isn't happening automatically
-          const typedData = data as
-            | CreateWorkspaceProjectMutation
-            | CreateProjectMutation
-
-          if (!typedData) return
-          const newProject =
-            'projectMutations' in typedData
-              ? typedData.projectMutations.create
-              : typedData.workspaceMutations.projects.create
-
-          if (newProject?.id) {
-            // Existing cache update for projects
-            modifyObjectFields<
```

Comment:
This cache mutation updates ROOT_QUERY.admin.projectList, but the new subscription or its cache updates dont do anything to replicate this behavior. If you remove this, this part is no longer gonna work.